libraries/protocol/include/scorum/protocol/config.hpp

```diff
@@ -141,8 +139,9 @@
 #define SCORUM_1_TENTH_PERCENT                 (SCORUM_100_PERCENT/1000)
 #define SCORUM_PERCENT(X)                      (uint16_t)(X*SCORUM_1_PERCENT)
 
-#define SCORUM_CONTENT_REWARD_PERCENT          (95*SCORUM_1_PERCENT)
-#define SCORUM_CURATION_REWARD_PERCENT         (25*SCORUM_1_PERCENT)
+#define SCORUM_DEV_TEAM_PER_BLOCK_REWARD_PERCENT    SCORUM_PERCENT(50)
```

Comment:
SCORUM_DEVELOPMENT_POOL_PER_BLOCK_REWARD_PERCENT

Reply:
Reward is aimed for dev team like(reward for witnesses and curators) not for dev pool